src/AutoRest.CSharp/Generation/Writers/MgmtLongRunningOperationWriter.cs

```diff
@@ -16,9 +16,9 @@
 
 namespace AutoRest.CSharp.Generation.Writers
 {
-    internal class LongRunningOperationWriter
+    internal class MgmtLongRunningOperationWriter
```

Comment:
There is a lot of duplication between this and management writer, can we share code? This file has a lot of reasonably complex code.

Reply:
I was following the pattern, sharing code in most of these places requires more templatization which didn't seem like a route that was desirable.  If we do this we can do it in a later PR